### PR TITLE
CIFF wasting treatment coverage updates

### PR DESCRIPTION
**Pull request by alibow**
These changes should not be implemented until after the calibration changes are made and reviewed

The coverage parameter (C) was split into C_MAM and C_SAM. Intervention scenario targets were updated. 

A strategy to account for the difference between coverage defined as percent of population vs. percent of population in need will need to be implemented but is not yet reflected in this PR.

### CODE REVIEW

**Review by aflaxman — APPROVED** [review]
One other point I have in my notes from that meeting is that instead of quickly going from low to high coverage, it would be useful to ramp up over 3 years.

File: docs/source/concept_models/vivarium_ciff_sam/concept_model.rst [hunk 1]
```diff
@@ -174,35 +174,38 @@ See linked documentation for more information :ref:`Treatment and management for
...
     - Sphere guideline for efficacy only
   * - MAM baseline
-    - 48.8% (95% CI: 37.4, 60.4)
+    - 15% (95% CI: 10, 20)
     - 73.1% (95% CI:58.5-87.7) for RUSF
-    - 0.488 x 0.731 = 0.34
-    - 1 - 0.34 = 0.66
-    - [Ackatia_Armah_2015]_
-    - Baseline coverage of MAM needs to be updated, efficacy comes from trial and may be too optimistic
+    - 15 x 0.731 = 0.11
```
Comment (NathanielBlairStahn) [inline docs/source/concept_models/vivarium_ciff_sam/concept_model.rst:186]:
```suggestion
    - 0.15 x 0.731 = 0.11
```

Comment (NathanielBlairStahn) [inline docs/source/concept_models/vivarium_ciff_sam/concept_model.rst:186]:
```suggestion
    - 0.15 x 0.731 = 0.11
```

File: docs/source/concept_models/vivarium_ciff_sam/concept_model.rst [hunk 2]
```diff
@@ -174,35 +174,38 @@ See linked documentation for more information :ref:`Treatment and management for
...
+    - 1 - 0.11 = 0.89
+    - Efficacy: [Ackatia_Armah_2015]_, Coverage: assumption from CIFF/UNICEF
+    - Efficacy comes from trial and may be too optimistic
   * - MAM alternative
-    - 90%
+    - 70%
     - 75% for RUSF
-    - 0.9 x 0.75 = 0.675
-    - 1 - 0.675 = 0.325
+    - 0.9 x 0.75 = 0.53
```
Comment (NathanielBlairStahn) [inline docs/source/concept_models/vivarium_ciff_sam/concept_model.rst:193]:
```suggestion
    - 0.7 x 0.75 = 0.53
```
Forgot to update to new value

File: docs/source/concept_models/vivarium_ciff_sam/concept_model.rst [hunk 3]
```diff
@@ -174,35 +174,38 @@ See linked documentation for more information :ref:`Treatment and management for
...
+    - 1 - 0.53 = 0.47
     - Sphere standards
     - Sphere guideline for efficacy only
```
Comment (NathanielBlairStahn) [inline docs/source/concept_models/vivarium_ciff_sam/concept_model.rst:196]: Where does the coverage assumption of 70% come from?

File: docs/source/gbd2020_models/risk_attributable_causes/wasting/index.rst [hunk 4]
```diff
@@ -644,35 +644,33 @@ in terms of the following variables:
...
      - With treatment and without death, average days spent in MAM before recovery
-     - 41.3
-     - Potentially to be updated
+     - 41.3 (95% CI: 34.4, 49)
+     - 
    * - :math:`time_to_sam_ux_recovery`
      - Without treatment or death, average days spent in SAM before recovery
-     - 60.5
-     - Potentially to be updated
+     - See the definition on the :ref:`wasting treatment intervention document <intervention_wasting_treatment>`
```
Comment (NathanielBlairStahn) [inline docs/source/gbd2020_models/risk_attributable_causes/wasting/index.rst:663]: Can you add a label at the correct place in the wasting treatment intervention document and then link to that so we know where to look?

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 5]
```diff
@@ -508,23 +514,23 @@ Deriving :math:`r_{SAM,ux}` using the following equations:
 
 #. :math:`r_{SAM,tx} = 1 / \text{time to recovery}_\text{effectively treated SAM}`
 
-#. :math:`1 / \text{time to recovery}_\text{overall SAM} =  1 / \text{time to recovery}_\text{effectively treated SAM} * C * E_{SAM} + 1 / \text{time to recovery}_\text{untreated SAM} * (1 - C * E_{SAM})`
+#. :math:`1 / \text{time to recovery}_\text{overall SAM} =  1 / \text{time to recovery}_\text{effectively treated SAM} * C_{SAM} * E_{SAM} + 1 / \text{time to recovery}_\text{untreated SAM} * (1 - C_{SAM} * E_{SAM})`
```
Comment (NathanielBlairStahn) [inline docs/source/intervention_models/wasting_treatment/index.rst:517]: This equation goes off the right side of my screen...

**Review by NathanielBlairStahn — APPROVED** [review]
Approving, but there a few things that should probably be fixed before merging (see comments above).

File: docs/source/concept_models/vivarium_ciff_sam/concept_model.rst (hunk 3 above)
Comment (alibow) [reply docs/source/concept_models/vivarium_ciff_sam/concept_model.rst:196]: This was suggested as a more realistic target by CIFF/UNICEF at our most recent meeting. I have added this as a note to the concept model, thanks.

File: docs/source/intervention_models/wasting_treatment/index.rst (hunk 5 above)
Comment (alibow) [reply docs/source/intervention_models/wasting_treatment/index.rst]: I split this up, thanks

File: docs/source/gbd2020_models/risk_attributable_causes/wasting/index.rst (hunk 4 above)
Comment (alibow) [reply docs/source/gbd2020_models/risk_attributable_causes/wasting/index.rst]: I am not sure we are set up in this repo to link to subsections on another page, or at least I couldn't figure out how. Instead I added a description of the appropriate section to find the relevant parameter by the name of the table of interest

Comment (NathanielBlairStahn) [reply docs/source/gbd2020_models/risk_attributable_causes/wasting/index.rst]: Sphinx lets you add cross-references to arbitrary locations:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations

The syntax is the same as the reference you have here that links to the label at the top of the page, but instead you just put a new label at some other location on the page and link to that.

Comment (alibow) [reply docs/source/gbd2020_models/risk_attributable_causes/wasting/index.rst]: Cool, done! Thanks

**Comment by NathanielBlairStahn** [comment]
I think it all looks good now, thanks Ali!

**Pull request merged**